content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -0,0 +1,66 @@
+---
+date: '2023-05-10'
+title: "Metrics"
+menu:
+  corda5:
+    parent: corda5-cluster-observability
+    identifier: corda5-cluster-metrics
+    weight: 2000
+section_menu: corda5
+---
+
+Metrics provide greater insight into the inner workings of Corda and can be used as the basis for monitoring and alerting.
+
+## Collecting Metrics
+
+All the Corda worker pods expose metrics in Prometheus text format at `/metrics` on port 7000.
+By default, this port is not exposed outside the Kubernetes cluster but most observability platforms support
+running an agent within the cluster that will dynamically detect Kubernetes pods exposing Prometheus endpoints and then poll for metrics.
+
+By default, the pods have the following Kubernetes annotations which may be sufficient for some monitoring agents
+to automatically scrape the endpoints:
+
+```yaml
+prometheus.io/scrape: "true"
+prometheus.io/path: "/metrics"
+prometheus.io/port: "7000"
+```
+
+These annotations can be disabled by providing the following overrides on the Corda Helm chart:
+
+```yaml
+metrics:
+    scrape: false
+```
+If you are using the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator),
+the Corda Helm chart supports the creation of a PodMonitor custom resource.
+The PodMonitor should be configured with the labels that the Prometheus Operator is set to discover.
+When using the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+Helm chart, this is the name of the Helm release for the Prometheus stack. For example:
+
+```yaml
+metrics:
+podMonitor:
+enabled: true
+labels:
+release: [RELEASE_NAME]
+```
+
+### Exported Metrics
+
+The following Corda-specific metrics are exported:
+
+{{</* table */>}}
+
+| Metric | Type | Description |
+| :----------- | :----------- | :----------- |
+| `corda_flow_run_time_seconds` | Timer | Time it took for a flow to complete successfully or to error. |
+
+{{</* /table */>}}
```

Comment:
Is this the right enclosing markup for tables ?

Reply:
This is what we have in our guidelined for Hugo shortcodes but I don't think it's right either. Updated table formatting and generating a preview to see how it look like on the site.

Reply:
I don't think it's necessary to call a shortcode for tables 🤔

Reply:
What did the table shortcode originally do, does anyone know (if anything)?

Reply:
Don't know - never used it!

Reply:
I tested the table and the tags are not necessary. Removed.